07-io-streams-and-files/snippets/src/bg/sofia/uni/fmi/mjt/io/ExecutionTimeComparator.java

```diff
@@ -7,76 +7,83 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 
-public class ExecutionTimeComparator {
+public class ExecutionTimeCompare {
```

Comment:
What I don't like about "ExecutionTimeCompare" for a class, name, it's a verb. I know that ExecutionTimeComparator may be misleading because of java.util.Comparator, still, looking for a better name. Maybe "ExecutionTimeExplorer"?

Reply:
IMHO ExecutionTimeComparator is OK. I did not change it intentionally :)